tests/grf/test_transformations.py

```diff
@@ -4,21 +4,25 @@
 
 import numpy as np
 
+import glass._array_api_utils as utils
```

Comment:
Elsewhere we have been using `_utils`

```suggestion
import glass._array_api_utils as _utils
```